sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileSourceCodecSuite.scala

```diff
@@ -71,6 +71,9 @@ class OrcCodecSuite extends FileSourceCodecSuite {
 
   override def format: String = "orc"
   override val codecConfigName: String = SQLConf.ORC_COMPRESSION.key
+  // Exclude "BROTLI" because its dependencies
+  // require adding different jars according to different OSs
```

Comment:
Which jar should be used when the user wants to enable this feature?

Reply:
This one. I excluded intentionally during upgrading to Apache ORC 2.0.0.

https://github.com/apache/spark/blob/681b41f0808e35313f4b63ca735f604d9ffc687f/pom.xml#L2615-L2618

Reply:
As he mentioned, native brotli should be available in the system, too.

Reply:
OK~

Reply:
Thanks for the comment, I've added the dependencies that need to be added to the PR description.